README.rst

```diff
@@ -88,10 +88,9 @@ from `here <https://github.com/HumanCellAtlas/data-store/blob/master/environment
 Then :code:`make test`.
 
 Primary CI testing is through Travis CI; there is also additional testing with the
-`Gitlab Allspark instance <https://allspark.dev.data.humancellatlas.org/HumanCellAtlas/dcp-cli/>`_ that runs tests for Windows.
-(Note that Allspark is not open to the public, members of the Human Cell Atlas project can access the Allspark cluster using the Github account
-associated with the Human Cell Atlas organization on Github.) If submitting PRs that have the potential of being platform-dependent, please ensure
-the status of "Windows Testing" is verified before merging.
+`Gitlab toilspark instance <https://ucsc-ci.org/DataBiosphere/data-store-cli>`_ that runs tests for Windows.
```

Comment:
we dont currently test windows on toilspark :0

Reply:
yeeeaaaaah, I was thinking about this. Lon mentioned it took quite a bit of work to get the windows runners set up properly. His meaning, I think, was "We did all this work to set them up, so why not leave them running". But my interpretation was "this is yet another service that is time-consuming to set up, get right, and maintain."

Since we're our own customers I see no reason to maintain Window support and Windows runners. Unless you're opposed, I'll remove mention of this from the Readme.

Reply:
Resolved in a8125ea